### PR TITLE
[PPA]: setup angular application, update workspace config, switch to jest and install tailwindcss

### PR DESCRIPTION
**Pull request by federicocamacho**

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=827846
+  "recommendations": ["angular.ng-template"]
```
Comment (mauro1998) [inline .vscode/extensions.json]: can you also add these to workspace recommendations?
image

File: packages/ppa-angular-client/jest.config.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export default {
+    preset: 'jest-preset-angular',
+  };
```
Comment (mauro1998) [inline packages/ppa-angular-client/jest.config.ts]: format

File: packages/ppa-angular-client/server.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { APP_BASE_HREF } from '@angular/common';
```
Comment (mauro1998) [inline packages/ppa-angular-client/server.ts:1]: Is this file for server side rendering?

File: packages/ppa-angular-client/src/app/app.component.html [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+          <clipPath id="a"><path fill="#fff" d="M0 0h982v239H0z" /></clipPath>
+        </defs>
+      </svg>
+      <h1>Hello, {{ title }}</h1>
+      <p>Congratulations! Your app is running. 🎉</p>
+    </div>
+    <div class="divider" role="separator" aria-label="Divider"></div>
+    <div class="right-side">
+      <div class="pill-group">
+        @for (item of [
```
Comment (mauro1998) [inline packages/ppa-angular-client/src/app/app.component.html:234]: is this like a new syntax for ngFor in the new versions?
I kind of like it.

**Review by mauro1998 — CHANGES_REQUESTED** [review]

File: packages/ppa-angular-client/server.ts (hunk 3 above)
Comment (federicocamacho) [reply packages/ppa-angular-client/server.ts:1]: yes it's created by default by the angular cli when you turn on ssr

File: .vscode/extensions.json (hunk 1 above)
Comment (federicocamacho) [reply .vscode/extensions.json]: added, altho I don't like it also includes other things like themes

File: packages/ppa-angular-client/jest.config.ts (hunk 2 above)
Comment (federicocamacho) [reply packages/ppa-angular-client/jest.config.ts]: done ty

File: .vscode/extensions.json (hunk 1 above)
Comment (mauro1998) [reply .vscode/extensions.json]: That's fine because it's optional, I don't like it either.

File: packages/ppa-angular-client/server.ts (hunk 3 above)
Comment (mauro1998) [reply packages/ppa-angular-client/server.ts:1]: I didn't know about this, I'll take a look at the docs when I get a chance

**Review by mauro1998 — APPROVED** [review]

**Pull request merged**